### PR TITLE
Adding preface in Style Guide

### PR DESCRIPTION
**Pull request by fmassey**

### CODE REVIEW

File: docs/styleguide.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+S.O.N.I.A C++ Style Guide
```
Comment (tmattio) [inline docs/styleguide.md:1]: I would write S.O.N.I.A. with the dot.

**Comment by tmattio** [comment]
Perfect preface, maybe just the typo in the title.

File: docs/styleguide.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+S.O.N.I.A C++ Style Guide
+=========================
+
+Preface
+-------
+
+C ++ is the main language in the S.O.N.I.A AUV platform.
```
Comment (tmattio) [inline docs/styleguide.md:7]: Same typo in S.O.N.I.A.

File: docs/styleguide.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+C ++ is the main language in the S.O.N.I.A AUV platform.
+The platform has been developed and maintained for many years by different members of the software team.
+This involves that programming styles may differ from project to project.
+With the recent desire to contribute to open-source movement, a standardization and an uniformity of style should be established.
+
+The purpose of this guide is to document procedures and best practice recommended at S.O.N.I.A.
+This document is not intended as a barrier to development, but as tool to improve the readability and maintainability of the code.
+
+Like C ++, this document evolves over time. Members are therefore invited to participate in the development of this document as well as its update.
```
Comment (des0ps) [inline docs/styleguide.md]: C++

**Pull request closed without merging**